src/System.Private.Uri/tests/UnitTests/Fakes/FakeUriParser.cs

```diff
@@ -6,5 +6,11 @@ namespace System
 {
     public class UriParser
     {
+    	internal static bool DontEnableStrictRFC3986ReservedCharacterSets
+        {
```

Comment:
Nit: spacing is not correct.

Reply:
Will be fixed when you fix the tab char.

---
In reply to: [158154555](https://github.com/dotnet/corefx/pull/26006#discussion_r158154555) [](ancestors = 158154555)